cmd/cli/commands/deals.go

```diff
@@ -234,40 +235,78 @@ var dealQuickBuyCmd = &cobra.Command{
 	},
 }
 
+func getBlacklistType() (pb.BlacklistType, error) {
+	switch blacklistTypeStr {
+	case "none":
```

Comment:
Why `none`, not `nobody` as it described in our proto?

Reply:
this was leaved as it was previous in the code - just reusble method was extracted